.github/workflows/build.yml

```diff
@@ -0,0 +1,29 @@
+on:
+  push:
+    branches: [master]
```

Comment:
Since this code will only stay in the parachain branch, should we add a trigger for it?